docs/docs/apps.md

```diff
@@ -22,3 +22,9 @@ Applications using gir.core. To add a new application to the list [edit this pag
 [Cavalier](https://github.com/NickvisionApps/Cavalier) is an audio visualizer based on CAVA.
 
 ![Cavalier screenshot](https://raw.githubusercontent.com/NickvisionApps/Cavalier/main/NickvisionCavalier.GNOME/Screenshots/Bars.png)
+
+## Pinta
+
+[Pinta](https://www.pinta-project.com/) is a image editing, drawing and painting application with a simple yet powerful interface
```

Comment:
```suggestion
[Pinta](https://www.pinta-project.com/) is a image editing, drawing and painting application with a simple yet powerful interface.
```

Can you add a `.` at the end of the sentence? Otherwise it looks good to me. Please amend your change to keep only one commit in your PR.